protocol-units/settlement/mcr/contracts/script/helpers/config.json

```diff
@@ -1,5 +1,5 @@
 {
-    "minDelay": "172800",
+    "minDelay": 172800,
```

Comment:
is this with respect to string-vs-integer value discussion?

Reply:
yes